examples/widgets/textbox.py

```diff
@@ -22,7 +26,9 @@
 
 
 def submit(text):
-    ydata = eval(text)
+    # user can enter a new math expression that uses "t" as its independent 
```

Comment:
You can turn this into a docstring. Also you can rename the function and the variable to make it more clear.

~~~
def submit_formula(expression):
    """
    Update the plotted function to the math *expression*.

    *expession*  is a string using "t" as its independent variable, e.g. "t **3".
    """
~~~